src/elements/OcSpinner.vue

```diff
@@ -33,51 +33,31 @@ export default {
      */
     size: {
       type: String,
-      default: "medium",
-      validator: value => {
-        return value.match(/(small|medium|large|x-large|xx-large)/)
-      },
+      default: null,
+    },
+  },
+  computed: {
+    ratio() {
+      let sizes = {
+        small: 0.5,
```

Comment:
Sure. As I was creating them, I needed to check wich sizes made sense and which didn't so I hardcoded them for the time being. These values are a bit unintuitive since the calculation the ratio instead of actual pixel sizes. Let's see what's best here.